src/components/moj-banner/_banner.scss

```diff
@@ -0,0 +1,59 @@
+/* ==========================================================================
+   #BANNER
+   ========================================================================== */
+
+.moj-banner {
```

Comment:
Consider refactoring:

```
.moj-banner {
  ...

  &__icon {
    ...
  }

  &__message {
    ...
  }

  &__assistive {
    ...
  }
}
```
And so on...

Reply:
We decided at HMCTS that it was actually easier to read as it is and to lean on SASS minimally. One to discuss for later and maybe with GOV.UK Frontend peeps.